src/main/java/com/iridium/iridiumskyblock/listeners/PlayerMoveListener.java

```diff
@@ -50,7 +52,16 @@ public void onPlayerMove(PlayerMoveEvent event) {
 
         if (event.getTo().getY() < 0 & IridiumSkyblock.getInstance().getConfiguration().voidTeleport) {
```

Comment:
U can check with this
```suggestion
        if (event.getTo().getY() < 0 && IridiumSkyblock.getInstance().getConfiguration().voidTeleport && IridiumSkyblockAPI.getInstance().isIslandWorld(player.getWorld()) {
```

Reply:
@PeachesMLG @dlsf  did you think this is good? or better is to get world from event location?

Reply:
lol i forgot a bracket